modules/richaudienceBidAdapter.js

```diff
@@ -54,7 +54,8 @@ export const spec = {
         scr_rsl: raiGetResolution(),
         cpuc: (typeof window.navigator != 'undefined' ? window.navigator.hardwareConcurrency : null),
         kws: (!isEmpty(bid.params.keywords) ? bid.params.keywords : null),
-        schain: bid.schain
+        schain: bid.schain,
+        gpid: raiSetPbAdSlot(bid)
```

Comment:
Please update unit test to account for new `gpid` field.

Reply:
i have added the test for gpid, please you can validate this?

Reply:
@lksharma